lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -111,6 +111,7 @@ def self.execute_and_exit_on_fail(machine, ui, command)
 
       def self.execute_once(machine, ui, command)
         machine.communicate.sudo(command) do |_, data|
+          next if data.chomp.empty?
```

Comment:
So why _next_ and not _return_. This seems inconsistent with the rest of the code. The aim is to return a string, right? Would that not by side effect return _true_ or something.